src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbolBase.cs

```diff
@@ -70,7 +74,9 @@ protected sealed override void MethodChecks(DiagnosticBag diagnostics)
             }
         }
 
-        protected abstract ParameterListSyntax GetParameterList();
+#nullable enable
+        protected abstract ParameterListSyntax? GetParameterList();
```

Comment:
>ParameterListSyntax? [](start = 27, length = 20)

Why is this nullable?

Reply:
That was left over from before the code flow and realized we guarded against null during construction. Fixed.